src/test/java/org/apache/datasketches/memory/test/AllocateDirectWritableMapMemoryTest.java

```diff
@@ -63,7 +64,9 @@
   private static boolean isFileReadOnly(final File file) {
     try {
       return (boolean) IS_FILE_READ_ONLY.invoke(null, file);
-    } catch (Exception e) { throw new RuntimeException(e); }
+    } catch (final IllegalAccessException | IllegalArgumentException | InvocationTargetException e) { 
+      throw new RuntimeException(e); 
```

Comment:
Why would this be re-thrown as a RuntimeException instead of re-throwing the original e?

Reply:
It could be done that way.  I guess this has been my habit since not all exceptions are runtime exceptions (some are checked exceptions)
.   But the "e" carries all the exception information about the type as well as the stack trace anyway